### DISCUSSION TITLE
Markor update - v2.8.0

### DISCUSSION THREAD

**Discussion by gsantner** [gsantner/markor#1403] [General]
### v2.8 [Blog Post](https://gsantner.net/blog/2021/07/29/markor-v2.8-todotxt-dialog-contexts-projects-multi-selection.html?source=github)

- Reference to GitHub discussion on More page, by @gsantner
- Add multi-selection to todo.txt dialogs, by @harshad1 @gsantner
- In-content search support for encrypted files, #1388 by @opensource21
- Remove alternative todo.txt naming (tags/categories), by @gsantner

<hr /><em>This discussion was created from the release <a href='https://github.com/gsantner/markor/releases/tag/v2.8.0'>v2.8.0</a>.</em>

**Comment by aswedaw** [comment]
Why don't you publish the latest release as APK here?

**Reply by gsantner** [reply]
Because you can also get the APK from F-Droid directly.
